spikeinterface_gui/traceview.py

```diff
@@ -13,6 +14,94 @@
 #   * 
 
 class MixinViewTrace:
+
+    MAX_RETRIEVE_TIME_FOR_BUSY_CURSOR = 0.5  # seconds
+
+    def get_data_in_chunk(self, t1, t2, segment_index):
```

Comment:
Here you factorize the 2 functions in one. This is good.
Are you sure that they were exactly the same ?
I think there were subtul diffrences no ?